PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/com/microsoft/intellij/serviceexplorer/azure/database/actions/SqlDatabaseConnectDataSourceAction.kt

```diff
@@ -27,9 +27,8 @@ import com.microsoft.tooling.msservices.helpers.Name
 import com.microsoft.tooling.msservices.serviceexplorer.azure.database.sqldatabase.SqlDatabaseNode
 import com.microsoft.tooling.msservices.serviceexplorer.azure.database.sqlserver.SqlServerNode
 
-
 @Name("Connect to database")
-class ConnectDatabaseAction(private val databaseNode: SqlDatabaseNode)
+class SqlDatabaseConnectDataSourceAction(private val databaseNode: SqlDatabaseNode)
```

Comment:
This rename is not really part of functionality in this PR, but made sense to unify with other action names.